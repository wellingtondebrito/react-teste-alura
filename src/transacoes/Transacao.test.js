import React from 'react'
import Transacao from './Transacao'

import {render, screen} from '@testing-library/react'

describe('Componente de transação do extrato', () => {

    const {container} = render(<Transacao data="17/06/2022" tipo="saque" valor="20.00"/>)

    it('O snapshot do componente deve permanecer sempre o mesmo', () => {
      expect(container.firstChild).toMatchSnapshot()  
    })
})