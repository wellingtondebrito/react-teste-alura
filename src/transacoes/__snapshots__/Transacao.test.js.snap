// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação do extrato O snapshot do componente deve permanecer sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    17/06/2022
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
